docs/javascript/v2/how--to-guides/record-and-live-stream/frame-capture.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Frame capture
+nav: 7.3
+---
+
+Some use-cases (proctoring, KYC and healthcare) require taking snapshots of video frames. This can be achieved on the `video` element by using web platform APIs (see snippet below).
+
+```javascript
+export const createSnapshotFromCanvas = inputCanvas => {
+  const link = document.createElement("a");
+  link.download = "snapshot.png";
+  link.href = inputCanvas.toDataURL("image/png");
+  link.click();
+};
+
+export const createSnapshotFromVideo = inputVideo => {
+  const canvas = document.createElement("canvas");
+  canvas.width = inputVideo.videoWidth;
+  canvas.height = inputVideo.videoHeight;
+  const context = canvas.getContext("2d");
+  context.drawImage(inputVideo, 0, 0, canvas.width, canvas.height);
+  createSnapshotFromCanvas(canvas);
+};
+```
```

Comment:
Let's also add a button with a click handler calling `createSnapshotFromVideo`.

Reply:
should we also add react snippet? getting video reference will be slightly different.(using `useRef`)

Reply:
yes, please. can you share the snippet here? will make these changes

Reply:
```jsx
const VideoTile = () => {
  const ref = useRef();
 // code to render video
  return ( 
   <video ref={ref} />
   <button onClick={() => {
      createSnapshotFromVideo(ref.current);
   }}>Capture Screenshot</button>
  )
}

```